### PR TITLE
allow copying of baselines

### PR DESCRIPTION
**Pull request by beav**
This commit adds a new API endpoint. You can POST to
`/baselines/<UUID>?display_name=<name>` and a new baseline is created
based on `<UUID>`.

### CODE REVIEW

File: system_baseline/views/v0.py [hunk 1]
```diff
@@ -258,6 +260,32 @@ def create_baseline(system_baseline_in):
     return baseline.to_json()
 
 
+@metrics.baseline_create_requests.time()
+@metrics.api_exceptions.count_exceptions()
+def copy_baseline_by_id(baseline_ids, display_name):
+    """
+    return a list of baselines given their ID
```
Comment (ehelms) [inline system_baseline/views/v0.py]: Needs an update

**Comment by beav** [comment]
@ehelms updated

File: system_baseline/views/v0.py [hunk 2]
```diff
@@ -258,6 +260,32 @@ def create_baseline(system_baseline_in):
     return baseline.to_json()
 
 
+@metrics.baseline_create_requests.time()
+@metrics.api_exceptions.count_exceptions()
+def copy_baseline_by_id(baseline_ids, display_name):
```
Comment (ehelms) [inline system_baseline/views/v0.py]: I was thinking that if the expectation is a single baseline ID, that treating this as an array of baseline IDS seemed odd from a developer understanding the code.

Comment (beav) [reply system_baseline/views/v0.py]: I wanted the openapi definitions to all live in the same section "/baselines/{baseline_ids}". However doing that requires me to do weird stuff like this.

I'll update and put the verbs for single baselines in its own section.

**Comment by beav** [comment]
@ehelms updated

**Pull request merged**